This huge PR adds proper SEO to all articles in the docs for better search engine indexing (#162)
golemfactory/golem-docs: src/pages/docs/creators/python/examples/tools/self-hosted-vm-images.md

---

> title: Guide to Self-Hosting VM Images for Golem Developers 

Reviewer: ```suggestion
title: Examples of how to Self-Host VM Images
```

> pageTitle: How to Self-Host and Distribute Golem VM Images – A Developer's Tutorial 

Reviewer: ```suggestion
pageTitle: How to Self-Host and Distribute Golem VM Images – concise instructions
```
